### ISSUE TITLE
Quote text is never set

### ISSUE THREAD

**Issue by philgyford** [beatonma/django-wm#35]
`Webmention` has a `quote` field but, as far as I can tell, it's never set.

I assume it would be good to set it in `mentions.tasks.incoming_webmentions.process_incoming_webmention()` using a tweaked `_update_wm()` function, but I'm not sure if you have in mind what text the quote should be?

Maybe use BeautifulSoup to try a few ways of getting something representative from the source page? Maybe grab the `.e-content`'s content (falling back to`.h-entry`, then trying `<main>`, then trying the meta description?), and then use the HTML-stripped, start of that?

**Comment by beatonma** [comment]
> I'm not sure if you have in mind what text the quote should be?

Not really. I think I added it with the intention of the user (at the time, myself) manually picking whatever excerpt they liked and adding it in the admin pages.

I think it's worth exploring this though. Getting the right context for a mention might be tricky for a general case but your suggestions sound reasonable.

P.S. Thanks for this and the other recent issues - I'm focussing on another project at the moment but will get back to this asap to sort them out!

**Comment by beatonma** [comment]
(This one hasn't been touched yet but now that the more problematic issues are fixed in 3.0.0 I'll try and do something with it)
